src/amber/cli/templates/app/src/controllers/application_controller.cr.ecr

```diff
@@ -3,4 +3,9 @@ require "jasper_helpers"
 class ApplicationController < Amber::Controller::Base
   include JasperHelpers
   LAYOUT = "application.<%= @language %>"
+
+  # If static resource is not found then raise an exception
+  def index
```

Comment:
Don't think this will cause a failure, but it could be bad that all controller that inherit from application_controller will have an index action that raises a 404 when called. What do you think of moving this controller into amber core `Amber::Controller::Static < Amber::Controller::Base`. Then at least it doesn't clutter the project, until we figure out a way to get rid of it entirely.